.github/workflows/ibc-path-validation.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
           list-files: shell
           filters: |
             ibc:
-              - added|modified: '_IBC/*.json'
+              - added|modified: '**/_IBC/*.json'
```

Comment:
remove maybe also
https://github.com/archway-network/networks/blob/main/.github/workflows/ibc-path-validation.yaml#L29-L33

Reply:
Why those need to be removed?

Reply:
The pipeline does sparse checkout and testnets dir is not on dirs list for the checkout